docs/typescript/activities.md

```diff
@@ -102,13 +102,13 @@ const { longRunningActivity } = proxyActivities<typeof activities>({
 
 ### Activity Retry Policy
 
-You can set a `retry` policy with [RetryOptions](https://typescript.temporal.io/api/interfaces/worker.RetryOptions) that define how activity is retried in case of failure.
+You can set a `retry` policy with [RetryPolicy](https://typescript.temporal.io/api/interfaces/worker.RetryPolicy) that define how activity is retried in case of failure.
```

Comment:
this doesnt show up in the generated docs yet but anticipating that it will be updated...